clang/www/analyzer/annotations.html

```diff
@@ -126,7 +126,7 @@ <h3 id="cocoa_mem">Cocoa &amp; Core Foundation Memory Management
 Annotations</h3>
 
 <!--
-<p>As described in <a href="/available_checks.html#retain_release">Available
+<p>As described in <a href="https://clang.llvm.org/docs/analyzer/checkers.html">Available
 Checks</a>,
 -->
```

Comment:
This is just a comment, I think we can remove this completely. Also, the original URL contained an anchor tag, but the new one does not, so the two sides aren't actually equivalent.

Reply:
I am unsure which one to link to, so I opted for less but not misleading information with this edit.